src/app/post/components/post-page/post-page.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+// import { Observable, Subscription } from 'rxjs';
+import { Post } from '../../../shared/interfaces/post.interface';
+import { ActivatedRoute } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { BlogService } from 'src/app/shared/services/blog.service';
+
+
+@Component({
+  selector: 'app-post-page',
+  templateUrl: './post-page.component.html',
+  styleUrls: ['./post-page.component.scss'],
+})
+export class PostPageComponent implements OnInit, OnDestroy {
+  public id!: string | null;
+  public notFound!: boolean;
+  public post!: Post;
+
+  constructor(private blogService: BlogService, private route: ActivatedRoute, private http: HttpClient) {}
+
+  ngOnInit(): void {
+    this.id = this.route.snapshot.paramMap.get('id');
```

Comment:
Do we have a possibility to navigate from one post-page to another? For example: we are one post with id=1, inside this route we have a link to post with id=2

Reply:
for now, I don't have such a possibility, maybe in the future, we will have a possibility to jump to another current author post or something like this.